api/src/main/java/org/openmrs/api/impl/ConceptServiceImpl.java

```diff
@@ -1982,6 +1984,72 @@ public boolean hasAnyConceptAttribute(ConceptAttributeType conceptAttributeType)
 		return dao.getConceptAttributeCount(conceptAttributeType) > 0;
 	}
 
+	/**
+	 * @see org.openmrs.api.ConceptService#getConceptByNameOrIdOrMap(String conceptRef
+	 */
+	@Override
+	@Transactional(readOnly = true)
+	public Concept getConceptByNameOrIdOrMap(String conceptRef) throws APIException {
+		if (conceptRef == null) {
```

Comment:
Presumably, this should be something like `StringUtils.isBlank(conceptRef)`, which handles both `null` and the case where `conceptRef` is only whitespace.

Reply:
Yea, Sure this will be better.  it will even eliminate the code below `if (!conceptRef.isEmpty())`

Reply:
Hello @ibacher! i am using java 8, and it doesn't have the method isBank(). Should i install java 11, which supports isBlank() method, but i fear messing up my installations of OpenMRS.  
how about i do it the old way.
```suggestion
		if (conceptRef.trim().isEmpty()) {
```
       .....
}

Reply:
@jnsereko I was suggesting using the method from [`StringUtils` in commons-lang](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html), which is already included in OpenMRS.

Reply:
Thank you @ibacher i have tool a look at the its documentation.  it is really good.

Reply:
Thanks @ibacher for the guidance,  if you take a look at 
[ConceptServiceImpl.java](https://github.com/openmrs/openmrs-core/blob/master/api/src/main/java/org/openmrs/api/impl/ConceptServiceImpl.java#L71) on line 71, you will see that the `StringUtils` class is imported using the statement `import org.springframework.util.StringUtils;`  But the version of this version does not define the `isBlank(String)` method we need.   
I tried importing the version of [StringUtils](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html) using the statement `import org.apache.commons.lang3.StringUtils;`  The problem is that this version of the class provides the required `IsBlank()` method needed but does not define the `hasText(String)`
having both the importation statements is also causing conflict..
I have decided to use `if (StringUtils.trimWhitespace(conceptRef).isEmpty()) { ... }`
Is this ok with you, or do you have any other better option?